File: src/com/vimukti/accounter/mobile/InputType.java
package com.vimukti.accounter.mobile;

public class InputType {
	int type;
	private int inputType;
	private String name = "";
	private String value = "";

	public InputType(int inputType) {
		this(inputType, "", "");
	}

	public InputType(int inputType, String name) {
		this(inputType, name, "");
	}

	public InputType(int inputType, String name, String value) {
		this.setInputType(inputType);
		this.setName(name);
		this.setValue(value);
		this.type = 3;
	}

	public String getName() {
		return name;
	}

	public void setName(String name) {
		this.name = name;
	}

	public int getInputType() {
		return inputType;
	}

	public void setInputType(int inputType) {
		this.inputType = inputType;
	}

	public String getValue() {
		return value;
	}

	public void setValue(String value) {
		this.value = value;
	}
}

File: platforms/iPad/Accounter iOS 1.0.2/Accounter/InputType.m
//
//  InputType.m
//  Accounter
//
//  Created by Amrit Mishra on 11/21/11.
//  Copyright 2011 __MyCompanyName__. All rights reserved.
//

#import "InputType.h"


@implementation InputType

-(id)init{
    
    self = [super init];
    if (self) {
        name = [[NSString alloc]init];
        
        textValue = [[NSString alloc]init];
        
    }
    return self;
}


-(void)setInputType:(int)newType{
    type = newType;
}
-(int)getInputType{

    return type;
}


-(UIKeyboardType)getKeyboardType{
    
    
    UIKeyboardType keyType;
    switch (type) {
        case 1:
            keyType = UIKeyboardTypeDefault;
            break;
        case 2:
            keyType =UIKeyboardTypeNumbersAndPunctuation;
            break;
        case 3:
            keyType = UIKeyboardTypeNumbersAndPunctuation;
            break;
        case 4:
            keyType = UIKeyboardTypeDefault;
            break;
        case 5:
            keyType = UIKeyboardTypeEmailAddress;
            break;
        case 6:
            keyType = UIKeyboardTypeNumbersAndPunctuation;
            break;
        case 7:
            keyType =  UIKeyboardTypeURL;
            break;
        case 8:
            keyType = UIKeyboardTypeDecimalPad;
            break;
        default:
            keyType = UIKeyboardTypeDefault;
            break;
    }
    return keyType;
}


-(void)setName:(NSString*)newName{
    name = newName;
}
-(NSString*)getName{
    return name;
}

-(void)setTextValue:(NSString*)newTextValue{
    textValue = newTextValue;
}

-(NSString*)getTextValue{
    return textValue;
}

@end
